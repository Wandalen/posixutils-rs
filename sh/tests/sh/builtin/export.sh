export export_test_var=value
echo $?
awk 'BEGIN { print ENVIRON["export_test_var"] }'

export_test_var2=value2
export export_test_var2
echo $?
awk 'BEGIN { print ENVIRON["export_test_var2"] }'

export -p | grep export_test_var