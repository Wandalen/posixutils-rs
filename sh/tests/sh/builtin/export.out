0
value
0
value2
export export_test_var='value'
export export_test_var2='value2'
